src/js/views/content/console/console.js

```diff
@@ -39,12 +39,23 @@ class Console {
     }
   }
 
+  parseStack(stack) {
```

Comment:
This looks fun

Reply:
It's weird to me the stack is only available as a big string. Plenty of functions/libraries out there but they were huge so this is a simple stupid version.